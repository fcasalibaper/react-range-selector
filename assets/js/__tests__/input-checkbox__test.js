/* eslint-disable */

jest.unmock('../src/input-checkbox');

import React from 'react';
import ReactDOM from 'react-dom';
import TestUtils from 'react-addons-test-utils';
import Checkbox from '../src/input-checkbox';

describe('CheckboxWithLabel', () => {

  it('changes the text after click', () => {
    const checkbox = TestUtils.renderIntoDocument(
      <Checkbox stateClass={'is-in-range'} />
    );

    const checkboxNode = ReactDOM.findDOMNode(checkbox);

    expect(checkboxNode.className).toEqual('range-selector__item-input is-in-range');

  });

});
