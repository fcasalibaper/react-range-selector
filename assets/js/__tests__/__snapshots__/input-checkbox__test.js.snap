// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="range-selector__item-input is-in-range"
>
  <input
    checked={undefined}
    className="range-selector__item-input-checkbox"
    id={undefined}
    onChange={[Function]}
    type="checkbox"
    value={undefined}
  />
  <label
    className="range-selector__item-input-text"
    htmlFor={undefined}
  />
</div>
`;
